src/Sylius/Behat/Page/Shop/HomePageInterface.php

```diff
@@ -38,4 +38,6 @@ public function getAvailableLocales(): array;
     public function switchLocale(string $localeCode): void;
 
     public function getLatestProductsNames(): array;
+
+    public function getCartItemsCount(): int;
```

Comment:
Is this method needed as the same one (looking at the content) has been added to `CartWidgetElementInterface`?